src/sankeyDiagram.ts

```diff
@@ -1757,6 +1762,10 @@ export class SankeyDiagram implements IVisual {
                     return SankeyDiagram.createLink(link);
                 }
             )
+            .attr("role", "option")
+            .attr("tabindex", 0)
+            .attr("aria-selected", sankeyDiagramUtils.isDataPointSelected)
+
```

Comment:
Need to add `aria-label` for desccriptive output

Reply:
Link ID is not for users, don't expose it to UI:

![image](https://github.com/microsoft/powerbi-visuals-sankey/assets/10897951/dd311da8-c1b3-439a-95b0-edc4c5333878)